src/sidebar/services/groups.ts

```diff
@@ -478,4 +480,52 @@ export class GroupsService {
       userid: 'me',
     });
   }
+
+  /**
+   * Fetch members for a group from the API and load them into the store as the
+   * focused group members.
+   */
+  async loadFocusedGroupMembers(): Promise<void> {
+    const groupId = this._store.focusedGroupId();
+    if (!groupId) {
+      return; // TODO Throw error?
+    }
```

Comment:
The store's `loadFocusedGroupMembers` method already checks that a group is focused before its members are set, and throws an error otherwise, so I thought maybe we don't need to throw a similar error in two different places.

On the other hand, if one should silently skip, maybe it should be the store and not the service, and an error should be thrown here.

Reply:
I ended up refactoring this, so my comment is not relevant anymore.